__test__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { CreateCustomerForm } from '@/components/form/CustomerForm';
+import sinon, { spy } from 'sinon';
+import { Customer } from '@/lib/types/apiTypes';
+import { ApiUpdateResponse } from '@/lib/types/hooks';
+
+const customerEndpointURL = `${process.env.NEXT_PUBLIC_API_URL}/customer`;
+const bodySpy = sinon.spy((body) => body);
+const pathSpy = sinon.spy((path) => path);
+
+jest.mock('@/lib/utils/fetch', () => ({
+    post: async (path: string, body: object): Promise<ApiUpdateResponse<Customer>> => pathSpy(path) && bodySpy(body),
+}));
+
+const testCustomerRequiredFields: Customer = { name: 'Name' };
+const testCustomerAllFields: Customer = { name: 'anotherName', description: 'description' };
+
+afterEach(() => {
+    bodySpy.resetHistory();
+    pathSpy.resetHistory();
+});
+
+test('a customer with the required fields only can be submitted', async () => {
+    render(<CreateCustomerForm />);
+
+    const nameInput = screen.getByTestId('form-field-name');
+    fireEvent.change(nameInput, { target: { value: testCustomerRequiredFields.name } });
```

Comment:
Find name field by the data-testid attribute, that is set here:

https://github.com/three-consulting/epoc-frontend/blob/390347c6e069404141393b21ffca3792e1ce518c/components/form/CustomerForm.tsx#L43-L52

Then set that to the value `testCustomerRequiredFields.name`.